js/views/pageNavVw.js

```diff
@@ -131,10 +131,21 @@ module.exports = Backbone.View.extend({
       var languageList = new window.List('homeModal-languageList', {valueNames: ['homeModal-language'], page: 1000});
       self.initAccordion('.js-profileAccordion');
 
-      // pre-select timezone
+      // pre-select local timezone and scroll it to the top.
       var offset = new Date().getTimezoneOffset();
       offset = '(GMT ' + (offset < 0 ? '+' : '-') + parseInt(Math.abs(offset/60)) + ':00)';
-      $("[id*='" + offset + "']").prop('checked', true);
+      var currentTimezone = $("[id*='" + offset + "']");
+      currentTimezone.prop('checked', true);
+      var timezoneList_ul = currentTimezone.parent().parent().parent();
```

Comment:
Use this.$('.js-homeModal-timeZoneList') as the selector instead. That way it doesn't break if the template hierarchy is changed, and "this.$" is the equivalent of "this.$el.find" which will limit the selector to the current view, just in case someone accidentally uses the same class in another view, or another copy of this view is present.